src/constants/tokens.ts

```diff
@@ -13,8 +13,11 @@ const size = {
 
 const zIndex = {
   backdrop: -1,
-  drawer: 10,
-  modal: 20,
+
+  // Set these values to 1 to utilize LeafyGreen's built-in stacking context.
```

Comment:
Is this a new change? (please share I must have missed the announcement)

Reply:
Yep, I mentioned this change in the PR description! Spruce already has these same values so they're just copied over from there 🤠